### PR TITLE
core-svc-feat(unleash): init unleash

### PR DESCRIPTION
**Pull request by ayocodingit**
## Overview
- core-svc-feat(unleash): init unleash

## Evidence:
project: Portal Jabar
title: core-svc-feat(unleash): init unleash
participants: @ayocodingit @rachadiannovansyah @iqbal167

### CODE REVIEW

File: core-service/src/cmd/main.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ func main() {
 	apm := utils.NewApm(cfg)
 	conn := utils.NewDBConn(cfg)
 	logrus := utils.NewLogrus()
+	utils.InitUnleash(cfg)
 
 	defer func() {
 		err := conn.Mysql.Close()
```
[the remark this answers is not in the document]
Comment (ayocodingit) [reply core-service/src/cmd/main.go:22]: I read documentation from unleash golang, for exit connection not required

[the remark this answers is not in the document]
Comment (ayocodingit) [reply core-service/src/cmd/main.go:22]: The Unleash we can connecting on run app, so each module can to use unleash anywhere

[the remark this answers is not in the document]
Comment (rachadiannovansyah) [reply core-service/src/cmd/main.go:22]: i see, that is great go straightforward 🚀

**Review by rachadiannovansyah — APPROVED** [review]

**Review by iqbal167 — APPROVED** [review]

**Comment by khihadysucahyo** [comment]
cool! 😘

**Pull request merged**